dags/debug.py

```diff
@@ -5,7 +5,7 @@
 default_args = {
     'owner': 'nobody@example.com',
     'depends_on_past': False,
-    'start_date': datetime(2099, 5, 31),
+    'start_date': datetime.now() - timedelta(3),
```

Comment:
Is it intentional to include this change in this PR? Do you think there's any danger here for prod?

Reply:
Yes, the simplest way to verify backfill behavior is to turn this DAG on. 

I don't think there should be any danger on production, this DAG should be turned off there (and state is maintained between deployment). The worst case I can imagine is where all the SleepOperators use up the execution slots for scheduling other things.